app/src/main/java/com/internship/move/MyApp.kt

```diff
@@ -0,0 +1,29 @@
+package com.internship.move
+
+import android.app.Application
+import com.internship.move.di.repositories
+import com.internship.move.di.storage
+import com.internship.move.di.viewModels
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+
+class MyApp : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        initKoin()
+    }
+
+    private fun initKoin() {
+        startKoin {
+            androidContext(this@MyApp)
+            modules(
```

Comment:
I believe there is a `modules(..)` overload that also accepts `vararg` instead of a list. That way you can simply enumerate the modules inside the call instead of instantiating a list
```
modules(
    viewModels,
    repositories,
    storage
)
```

Vararg comes from "variable number of arguments", more info here
https://kotlinlang.org/docs/functions.html#variable-number-of-arguments-varargs